no-computer.md

```diff
@@ -1,43 +1,51 @@
 good way to meet their needs.
 
 
+
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
+The need is for communication between teachers and students about what work needs to be done in order to succeed in the class. Problem sets help both teachers and students in quantitative courses by providing them with means to enumerate what tasks need to be done for homework and how they should be done. Without problem sets math students would be left trying to find good problems to help them learn the material while professors would be forced to deal with students that were all doing different work. With problem sets it helps professors and graders be of more help to the students as well because they can be very familiar with the problems that students are doing and therefore be able to provide tips without giving away the answer or disallowing the students from learning.
 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
+A language meets their needs because it allows the professor to tell all of his students exactly what to do, have them understand exactly what needs to be done, and do it in a specific way that is understandable to the professor, grader, and other students.
+
 
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
+It excites me because I do matjh problem sets almost every day and dont know how i would learn the material without them. Also, without problem sets I wouldnt be able to get specific help so it is really relevant to me. I came up with it when I was using latex to write a problem set.
 
 
 ### Domain
 _Describe the project's domain in five words._
+Students and professors of quantitatives.
 
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
-
+The user in this case is the professor who writes the program (the PS) that is then executed by the computer (the student). Programming looks like making up problems or selecting them from a problem book and assigning them to the students. If you dont do it in this way, you really arent using the language at all!
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
+When a program runs a student does the problems in the program and explains how he did them. A program interacts with the user by being graded and errors might occur if the student misunderstood the problem, made a computational answer, could not solve the problem, or was lazy. This is communicated in the output of the program but is not overtly identified as an error. Bad error messages in this lanugage!!
```

Comment:
wait, but errors would be in writing the language. Is the language writing the pset or solving the pset?